docs/layouts/partials/head.html

```diff
@@ -6,6 +6,9 @@
   <!--[if (gt IE 9)|!(IE)]><!--> <html class="no-js" {{ with .Site.LanguageCode }}lang="{{ . }}"{{ end }}> <!--<![endif]-->
 
   <head>
+    <script type="text/javascript">
+      !function(){if("function"!=typeof window.bugyard){var a=function(){a.c(arguments)};a.q=[],a.c=function(b){a.q.push(b)},window.bugyard=a;var b=document.createElement("script");b.setAttribute("data-bugyard","61e0ec1038569600142ffcd2"),b.setAttribute("async","async"),b.setAttribute("defer","defer"),b.setAttribute("src","https://widget.bugyard.io/bugyard.min.js"),document.getElementsByTagName("head")[0].appendChild(b)}}();
+    </script>
```

Comment:
All other scripts are down around #L47 -- should this be there, as well, or does it need to be up top here?

Also, let's comment this one out until we have Bugyard approved for use in production. :)

```suggestion
    <!--
    <script type="text/javascript">
      !function(){if("function"!=typeof window.bugyard){var a=function(){a.c(arguments)};a.q=[],a.c=function(b){a.q.push(b)},window.bugyard=a;var b=document.createElement("script");b.setAttribute("data-bugyard","61e0ec1038569600142ffcd2"),b.setAttribute("async","async"),b.setAttribute("defer","defer"),b.setAttribute("src","https://widget.bugyard.io/bugyard.min.js"),document.getElementsByTagName("head")[0].appendChild(b)}}();
    </script>
    -->
```